src/forwarder/Forwarder.sol

```diff
@@ -43,7 +43,7 @@ contract Forwarder is IForwarder, Ownable, Nonces, Multicall, SelfPermit, Reentr
     /// ------------------------------- Constants -------------------------------
 
     bytes private constant FORWARDREQUEST_TYPE = abi.encodePacked(
-        "ForwardRequest(address user,address to, address paymentToken, bytes data,uint256 deadline,uint256 nonce)"
+        "ForwardRequest(address user,address to,address paymentToken,bytes data,uint256 deadline,uint256 nonce)"
```

Comment:
the previous commit used uint64 for deadline

Reply:
good catch!